### DISCUSSION TITLE
Cant upload proof of my academic status

### DISCUSSION THREAD

**Discussion by gucio321** [community/community#104249] [GitHub Education]
### Hi everyone,

Yesturday GitHub asked me to renew my student pack. As always I was asked to upload a phot of my academic ID. Unfortunately I can't upload the photo anymore - I need to take it with my phone and upload directly to GItHub's website.
The problem is that my browser doesn't want to switch my phone camera to back camera and the front camera has too low quality. Is it any way to upload the photo as usual?

**Comment by gucio321** [comment]
I've tried on the other laptom and it even doesn't see its camera in the browser.

I hope this will be fixed by GitHub soon...

**Comment by JCCourt** [comment]
Hi, gucio

I posted this answer in a different thread but I don't know if it will help or not. Let me know if it does or not.


* Apply via smartphone

I think this has been a known issue for a while now. I also had the same problem uploading my files via PC. But once I tried applying on my phone I had no problems. If for whatever reason smartphone doesn't work, refer to the [supported browsers page](https://docs.github.com/en/get-started/using-github/supported-browsers) for trouble shooting.

If this helped. Please "Mark as Answered" so future users can see this information to help them too 😊

-- JCCOURT

**Reply by gucio321** [reply]
Unfortunately using my smartphone didn't help. I can't switch my camera do that i can use only the front one. If i try to use the other camera with the better quality i see black screen.

I'm using brave browser (which is modified chromium) but i also tried chrome with the same result

**Comment by faridnec** [comment]
I had submitted my application previously with my laptop camera which does not have a really good quality as long as it is readable but then they approved it. For suggestion you probably can try using the available options and source

**Comment by gucio321** [comment]
Well i see there is no good solution to do this "as intended". Ill just take a photo normally and use OBS to make it a camera output

Btw which side of my id should i upload? This with my name or this with validity date? (They are two different sides in mY cases)

**Reply by faridnec** [reply]
I use the side which displays my name and on my ID there is not even the validity date to begin with lol yet they approved it. Give it a try

**Reply by gucio321** [reply]
ok, thats great! also, idk what I did (or mayge GitHub has fixed it already) but this appeared:
![image](https://github.com/community/community/assets/73652197/c152c900-8b6a-4f8b-ab4d-e13f101a5de1)

**Reply by faridnec** [reply]
i didn't use the upload method previously. I took my id using the camera there and upload it. I do not really remember, sorry

**Comment by mauriceoboya** [comment]
f you're facing difficulties uploading a photo directly from your phone to GitHub due to camera issues, there are a few alternative solutions you can try:
 Upload from Computer: If you have the photo saved on your computer, you can transfer it to your computer and upload it from there. Use a USB cable, email, cloud storage, or any other preferred method to transfer the photo.
Use a Different Browser: If the issue is browser-specific, try using a different browser to access GitHub. Some browsers may have better compatibility with certain features, and switching browsers might resolve the camera issue.
 Contact GitHub Support:If none of the above solutions work, consider reaching out to GitHub support for assistance. They may provide specific guidance or offer a solution to help you upload the required photo.

**Reply by gucio321** [reply]
problem was that i couldn't "upload" the photo. I had to take it with my camera shared to my browser (for some reason). now it changed and I managed to upload my proof. Thank you all for help.
